backend/.env.example

```
@@ -1,9 +1,16 @@
 PORT=3001
 DATABASE_URL="postgresql://<DB_USER>:<DB_PASSWORD>@localhost:5432/<DB_NAME>"
 CORS_ORIGIN=http://localhost:3000
+FRONTEND_URL=http://localhost:3000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The FRONTEND_URL key should go before the PORT key

(UnorderedKey)